jwst/straylight/straylight.py

```diff
@@ -182,5 +182,17 @@ def correct_xartifact(input_model, modelpars):
     model[:, 0:4] = 0.0
     output.data = output.data - model
 
+    # Remove any remaining pedestal stray light values based on inter-channel region
+    # chosen based on analysis of multiple bands of data from various programs
+    if (channel == '12'):
+        pedestal = np.nanmedian(output.data[280:740, 503:516])
```

Comment:
My typical reaction to hardcoded values like is not a good one, but if you're really sure that these will work universally, I'm OK with it.